### PR TITLE
Automatically extract CN and TW characters from the translation file.

### PR DESCRIPTION
**Pull request by zyren**
I created get_cn_char.py and get_tw_char.py to extract all Chinese characters from the translation file. I modified make_fonts.sh to import the extraction results, making it very easy to generate the font library.

### CODE REVIEW

**Comment by arthurson** [comment]
any chance to change font to more readible?
now it is hard to read the chinese.

**Comment by zyren** [comment]
> 

May I ask if the Chinese expressions are hard to understand, or the font is not pleasing? If some translations are inaccurate, please correct them. You can list some confusing contents, and I will do my best to check and modify to achieve the best.

**Comment by richardclli** [comment]
Let me take a look.

**Comment by arthurson** [comment]
i mean the font now it uses is not clear to read, not related to translations that you made.

File: radio/src/fonts/lvgl/get_cn_char.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+def extract_chinese_characters(file_path):
+    chinese_characters = set()
+    
+    with open(file_path, 'r', encoding='utf-8') as file:
+        for line in file:
+            for char in line:
+                if '\u4e00' <= char <= '\u9fff':
```
Comment (raphaelcoeffic) [inline radio/src/fonts/lvgl/get_cn_char.py:7]: Please define a "constant" for these, so we know what they correspond to.

**Review by raphaelcoeffic — CHANGES_REQUESTED** [review]
It seems these 2 files are more or less copy of one another. Can we have a consolidated version using parameters so that difference can be worked out?

**Comment by richardclli** [comment]
Replaced by https://github.com/EdgeTX/edgetx/pull/4202

**Pull request closed without merging**